code/modules/mob/mob.dm

```diff
@@ -300,6 +300,8 @@
 	face_atom(A)
 	A.examine(src)
 
+/atom/proc/pointed_at(var/mob/user)
```

Comment:
absolute state of hippie coder lol

Reply:
he's a hippie coder? i thought he was someone from para or citadel.